packages/common/random-access-storage/src/common/storage.ts

```diff
@@ -9,6 +9,9 @@ export enum StorageType {
   RAM = 'ram',
   IDB = 'idb',
   CHROME = 'chrome',
```

Comment:
what is "chrome" storage?

Reply:
Doesn't seem to be used, we can deprecate this too